### ISSUE TITLE
Framerate drops when using mouse or keyboard on Windows.

### ISSUE THREAD

**Issue by quietfanatic** [TASEmulators/desmume#511]
When I'm using the latest build, the performance is great, except that when I use keyboard input or move the mouse over the window, the framerate tanks, sometimes as low as 12fps.  I've tried changing all sorts of emulation and display settings to no avail, which makes me think something's going weird in the input stack.  No extra CPU is being used during the slowdown according to my task monitor (if anything, CPU usage is lower).

The stable v0.9.11 release works just fine.

[Some extra information in case it matters (probably doesn't)]
Machine: Lenovo thinkpad x1 yoga 3rd gen or so
OS: Windows 11 build 22563
Emulator build date: 2022-02-12
Software: Sekaiju no Meikyuu

**Comment by zeromus** [comment]
the problem is your shoddy GPU drivers.
change config > display method

**Comment by quietfanatic** [comment]
Thanks, I thought I had my power settings set to maximum performance, but it must have reverted to balanced during some update, or I changed it without remembering.  Changing it back to maximum performance seems to have completely fixed the problem.

For what it's worth, while investigating I found that reverting 92924db13613ad315f4d7ea509eacf48b0a3caf6 and 8d011f834432d1d08ca5910f687a249df973c8cd seemed to greatly improve the problem under balanced power settings, though not fix out outright.  Moving the mouse vigorously still caused a slight slowdown (just dropping a couple fps occasionally) but slight movements and keyboard inputs seemed to cause no problem.

I'm keeping this issue open in case you want to investigate those commits, but if you decide it's not worth investigating go ahead and close it.  Thanks again!
